### PR TITLE
Keras model import - updater lr fix

### PR DESCRIPTION
**Pull request by eraly**
Signed-off-by: eraly <susan.eraly@gmail.com>

## What changes were proposed in this pull request?
Bug Fix. With certain updaters the learning rate will not be applied correctly. In DL4J the learning rate is picked via the learning schedule and here the initial learning rate is set to 1. Correcting to the learning rate specified in the config.

## How was this patch tested?
Manually

## Quick checklist

The following checklist helps ensure your PR is complete:

- [X] Eclipse Contributor Agreement signed, and signed commits - see [IP Requirements](https://deeplearning4j.org/eclipse-contributors) page for details
- [X] Reviewed the [Contributing Guidelines](https://github.com/eclipse/deeplearning4j/blob/master/CONTRIBUTING.md) and followed the steps within.
- [X] Created tests for any significant new code additions.
- [X] Relevant tests for your changes are passing.

### CODE REVIEW

File: deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/utils/KerasOptimizerUtils.java [hunk 1]
```diff
@@ -75,7 +75,7 @@ public static IUpdater mapOptimizer(Map<String, Object> optimizerConfig)
                 dl4jOptimizer = new Adam.Builder()
                         .beta1(beta1).beta2(beta2)
                         .epsilon(epsilon).learningRate(lr)
-                        .learningRateSchedule(new InverseSchedule(ScheduleType.ITERATION, 1, decay, 1))
+                        .learningRateSchedule(new InverseSchedule(ScheduleType.ITERATION, lr, decay, 1))
```
Comment (AlexDBlack) [inline deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/utils/KerasOptimizerUtils.java]: Good catch on this. Though honestly: for the decay = 0 case, I think I'd rather not have the schedule object there at all...
So `.learningRate(lr).learningRateSchedule(decay == 0 ? null : new InverseSchedule(ScheduleType.ITERATION, lr, decay, 1))`

Comment (eraly) [reply deeplearning4j/deeplearning4j-modelimport/src/main/java/org/deeplearning4j/nn/modelimport/keras/utils/KerasOptimizerUtils.java]: Cleaned it up.
